{"name": "Check if the logged out user can navigate to Sign In", "status": "broken", "statusDetails": {"message": "selenium.common.exceptions.TimeoutException: Message: Element by locator ('css selector', \"a[data-test='accountNav-signIn']\") is not clickable\n\n", "trace": "  File \"/Users/mary/opt/anaconda3/lib/python3.9/site-packages/behave/model.py\", line 1329, in run\n    match.run(runner.context)\n  File \"/Users/mary/opt/anaconda3/lib/python3.9/site-packages/behave/matchers.py\", line 98, in run\n    self.func(context, *args, **kwargs)\n  File \"features/steps/header.py\", line 23, in click_nav_sign_in\n    context.app.header.wait_and_click(SIGN_IN_NAV)\n  File \"/Users/mary/Documents/QA Course/Careerist/Python and QA Automation/QA Automation/python-selenium-automation/pages/base_page.py\", line 54, in wait_and_click\n    self.wait.until(\n  File \"/Users/mary/opt/anaconda3/lib/python3.9/site-packages/selenium/webdriver/support/wait.py\", line 105, in until\n    raise TimeoutException(message, screen, stacktrace)\n"}, "steps": [{"name": "Given Open target main page", "status": "passed", "start": 1725837541178, "stop": 1725837545111}, {"name": "When Click Sign In", "status": "passed", "start": 1725837545112, "stop": 1725837545312}, {"name": "When From right side navigation menu, click Sign In", "status": "broken", "statusDetails": {"message": "selenium.common.exceptions.TimeoutException: Message: Element by locator ('css selector', \"a[data-test='accountNav-signIn']\") is not clickable\n\n", "trace": "  File \"/Users/mary/opt/anaconda3/lib/python3.9/site-packages/behave/model.py\", line 1329, in run\n    match.run(runner.context)\n  File \"/Users/mary/opt/anaconda3/lib/python3.9/site-packages/behave/matchers.py\", line 98, in run\n    self.func(context, *args, **kwargs)\n  File \"features/steps/header.py\", line 23, in click_nav_sign_in\n    context.app.header.wait_and_click(SIGN_IN_NAV)\n  File \"/Users/mary/Documents/QA Course/Careerist/Python and QA Automation/QA Automation/python-selenium-automation/pages/base_page.py\", line 54, in wait_and_click\n    self.wait.until(\n  File \"/Users/mary/opt/anaconda3/lib/python3.9/site-packages/selenium/webdriver/support/wait.py\", line 105, in until\n    raise TimeoutException(message, screen, stacktrace)\n"}, "start": 1725837545312, "stop": 1725837560338}, {"name": "Then Verify Sign In form opened", "status": "skipped", "start": 1725837560540, "stop": 1725837560540}], "start": 1725837538586, "stop": 1725837560540, "uuid": "7113e4f7-50c4-4b5e-9c6f-2c3b6aa3f941", "historyId": "7257318c220f337248ad44b204229878", "fullName": "Main page tests: Check if the logged out user can navigate to Sign In", "labels": [{"name": "severity", "value": "normal"}, {"name": "tag", "value": "smoke"}, {"name": "feature", "value": "Main page tests"}, {"name": "framework", "value": "behave"}, {"name": "language", "value": "cpython3"}]}